misc/misc.go

```diff
@@ -13,3 +26,57 @@ func MinMax(array []int) (int, int) {
 	}
 	return min, max
 }
+
+func Portcheck() (int, error) {
+	min, endingPort := minMax(config.Cfg.Common.Ports)
+	for startingPort := min; startingPort < endingPort; startingPort++ {
```

Comment:
While checking your suggestion, I also noticed there is an Off-by-one error. I don't see a reason why we shouldn't include the `endingPort` too.